sample/src/main/java/com/canhub/cropper/sample/camera_java/presenter/SCameraPresenterJava.java

```diff
@@ -0,0 +1,169 @@
+            view.showErrorMessage("onCreate activity and/or context are null");
+            return;
+        }
+        this.context = context;
+
+        request = ActivityCompat.shouldShowRequestPermissionRationale(activity, Manifest.permission.CAMERA);
+        assert context.getPackageManager()!=null;
+        hasSystemFeature = context.getPackageManager().hasSystemFeature(PackageManager.FEATURE_CAMERA_ANY);
+        selfPermission = ContextCompat.checkSelfPermission(context, Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED;
+    }
+
+    @Override
+    public void startWithUriClicked() {
+        assert view!=null;
+        if(hasSystemFeature && selfPermission){
+            view.dispatchTakePictureIntent();
+        } else if(hasSystemFeature && minVersion && request){
+            view.showDialog();
+        } else if(hasSystemFeature){
+            view.cameraPermissionLaunch();
+        } else {
+            view.showErrorMessage("onCreate no case apply");
+        }
+    }
+
+    @Override
+    public void startWithoutUriClicked() {
+        assert view!=null;
+        view.startCropImage(CameraEnumDomainJava.START_WITHOUT_URI);
+    }
+
+    @Override
+    public void startPickImageActivityClicked() {
+        assert view!=null;
+        view.startCropImage(CameraEnumDomainJava.START_PICK_IMG);
+    }
+
+    @Override
+    public void startActivityForResultClicked() {
+        assert view!=null;
+        view.startCropImage(CameraEnumDomainJava.START_FOR_RESULT);
+    }
+
+    @Override
+    public void onOk() {
+        assert view!=null;
+        view.cameraPermissionLaunch();
+    }
+
+    @Override
+    public void onCancel() {
+        assert view!=null;
+        view.showErrorMessage("onCancel");
+    }
+
+    @Override
+    public void onActivityResult(int resultCode, int requestCode, Intent data) {
+        assert view!=null;
+        if (resultCode == RESULT_OK) {
+            switch (requestCode) {
+                case CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE : {
+                    assert(context!=null);
+                    Bitmap bitmap = CropImage.getActivityResult(data).getBitmap(this);
+                    Log.v(
+                            "File Path",
+                            CropImage.getActivityResult(data).getUriFilePath(this).toString()
+                    );
+
+                    Uri uriContent = CropImage.getActivityResult(data).getUriContent();
+                    if(uriContent!=null) {
+                        view.handleCropImageResult(uriContent.toString().replace("file:", ""));
+                    } else {
+                        view.showErrorMessage("CropImage getActivityResult return null");
+                    }
+                    break;
+                }
+                case SCameraFragmentJava.CUSTOM_REQUEST_CODE : {
+                    assert context!=null;
+                    Log.v("File Path", CropImage.getPickImageResultUriFilePath(context, data));
```

Comment:
To remove the `getPickImageResultUriFilePath ` error, please add `@JvmStatic` above it method.

On class `CropImage.kt`:
```
@JvmStatic
fun getPickImageResultUriFilePath(
```